src/test/java/model/PlayerTest.java

```diff
@@ -0,0 +1,29 @@
+package model;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.assertj.core.api.Assertions;
+
+public class PlayerTest {
+
+    @Test
+    @DisplayName("Player 객체 생성 성공 테스트")
+    void createPlayerTest(){
+        Assertions.assertThatNoException().isThrownBy(()->{Player player = new Player(new Name("pobi"));});
```

Comment:
```suggestion
        Assertions.assertThatNoException().isThrownBy(() -> 
             new Player(new Name("pobi"))
        );
```

불필요한 로컬변수를 제거하고 컨벤션에 맞게 스페이스를 추가해도 가독성이 훨씬 좋아진답니다 ㅎㅎ 
개행을 반드시 저렇게 할 필요는 없지만 불필요한 것 제거 및 스페이스는 꼭 지키면 좋아요 😊